### DISCUSSION TITLE
Linux Build option?

### DISCUSSION THREAD

**Discussion by shawstar** [o3de/o3de#14504] [Help Wanted]
I just noticed that just recently the Linux build instructions are missing, but I recall where here last month in early January? 

https://github.com/o3de/o3de

**Comment by AMZN-alexpete** [comment]
The instructions for building the engine are here:
https://www.o3de.org/docs/welcome-guide/setup/setup-from-github/building-linux/

Copied example from that page of the configure/build steps (refer to the page above if you have not registered the engine before)
```bash
cmake -B build/linux -S . -G "Ninja Multi-Config" -DLY_3RDPARTY_PATH=$HOME/o3de-packages
cmake --build build/linux --target Editor --config profile -j 8
```

And for creating/building a project here:
https://www.o3de.org/docs/welcome-guide/create/creating-projects-using-cli/creating-linux/

```bash
cd $HOME/O3DE/Projects/MyProject
cmake -B build/linux -S . -G "Ninja Multi-Config"
cmake --build build/linux --target MyProject.GameLauncher Editor --config profile -j 8
```

**Reply by flindt** [reply]
The link does not seem to point to the right place.

![2023-06-17-164129_956x1041_scrot](https://github.com/o3de/o3de/assets/469086/887523e9-7ef5-4fd1-b293-62ce9f329484)

**Reply by AMZN-daimini** [reply]
> The link does not seem to point to the right place.

Documentation has been moved to https://docs.o3de.org/docs/welcome-guide/setup/setup-from-github/building-linux/
Redirects are not in place yet but will hopefully be up soon.
